pages/integrations/sso.md

```diff
@@ -96,6 +96,9 @@ Yes, team maintainers can select whether a user is 'required' to use SSO or whet
 ### Do you support JIT provisioning?
 Yes, we do.
 
+### What **is** JIT provisioning?
+Just In Time (JIT) provisioning is a method of creating accounts at the point in which they are needed; you can give a user access to Buildkite in your SSO provider, but their account won't be created (and thus they won't use a "seat") until it's required, which is assumed to be their first login attempt.
```

Comment:
General tightening:

```suggestion
Just-in-time user provisioning (JIT provisioning) creates accounts only when needed. You can grant a user access to Buildkite through your SSO provider, but their account won't be created until it's required—typically upon their first login attempt. For billing purposes, the user doesn't exist until their account is created.
```

Reply:
This suggests a reword, but I think it's better to use the suggestion below to include the text in the previous question heading.